appveyor.yml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
This file does not make sense, because we use Travis CI and as far as I know Caddy does not support Windows.